src/platform/packages/shared/response-ops/alerts-table/components/bulk_actions_toolbar_control.tsx

```diff
@@ -44,19 +42,24 @@ const selectedIdsToTimelineItemMapper = (
 ): TimelineItem[] => {
   return Array.from(rowSelection.keys()).map((rowIndex: number) => {
     const alert = alerts[rowIndex];
+    const data = Object.entries(alert).map(([key, value]) => ({
```

Comment:
nit: how about adding the default empty values for the well known fields here instead of iterating on the fields array each time?


```suggestion
    const data = Object.entries({
      [ALERT_CASE_IDS]: [],
      [ALERT_WORKFLOW_TAGS]: [],
      [ALERT_WORKFLOW_ASSIGNEE_IDS]: [],
      ...alert,
     }).map(([key, value]) => ({
```

Reply:
Apologies, missed this comment before I merged. Made a tiny pr to fix here: https://github.com/elastic/kibana/pull/237307